MonticelloTonel-Core.package/TonelWriter.class/instance/shouldWriteChangestamp..st

```diff
@@ -0,0 +1,5 @@
+private testing
+shouldWriteChangestamp: aString
+
+	^ aString notNil and: [aString notEmpty and: [
+			(self currentPackageProperties at: TonelWriter squeakChangestamp ifAbsent: [false]) = true]]
```

Comment:
I was wondering: for Squeak projects, this should be on by default, right? So should the ifAbsent: default to something `Smallltalk version`-specific?

Reply:
...but then it would also get written when Squeakers contribute to Pharo projects from Squeak. I see the problem. Is there a way we can turn this on safely for Squeak projects?

Reply:
This could be something that Squot is responsible for. When adding packages to a project it could create the corresponding package.st files and configure them accordingly. Ideally, it would also keep the setting consistent when adding further packages and offer UI to change the setting.

Reply:
Pharo projects will most likely also not want the Squot metadata files. So as a workaround, one could tie the timestamp setting to the metadata-writing setting.

Reply:
Let's continue this here: https://github.com/hpi-swa/Squot/issues/300